paper/paper.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'Efficient Polyhedral Gravity Modeling in Modern C++ and Python'
+tags:
+  - C++
+  - Python
+  - astronomy
+  - dynamics
+  - asteroids
+  - gravity
+  - numerical methods
+  - polyhedral model
+authors:
+  - name: Jonas Schuhmacher
+    orcid: 0009-0005-9693-4530
+    affiliation: 1
+  - name: Emmanuel Blazquez
+    orcid: 0000-0001-9697-582X
+    affiliation: 2
+  - name: Fabio Gratl
+    orcid: 0000-0001-5195-7919
+    affiliation: 1
+  - name: Dario Izzo
+    orcid: 0000-0002-9846-8423
+    affiliation: 2
+  - name: Pablo Gómez
+    orcid: 0000-0002-5631-8240
+    affiliation: 2
+affiliations:
+ - name: Chair for Scientific Computing, Technische Universität München, Arcisstraße 21, 80333 München, Germany 
+   index: 1
+ - name: Advanced Concepts Team, European Space Agency, European Space Research and Technology Centre (ESTEC), Keplerlaan 1, 2201 AZ Noordwijk, The Netherlands
+   index: 2
+date: 07 July 2023
+bibliography: paper.bib
+
+---
+
+# Summary
+
+Polyhedral gravity models are ubiquitous for modelling the gravitational field of irregular bodies, such as asteroids and comets. We present an open-source C++ library for the efficient, parallelized computation of a polyhedral gravity model. We also provide a Python interface to the library using *pybind11*. The library is particularly focused on delivering high performance and scalability, which we achieve through vectorization and parallelization with *xsimd* and *thrust*, respectively. The library supports many common formats, such as *.stl*, *.off*, *.ply*, *.mesh* and *tetgen*'s *.node* and *.face*.
+
+# Statement of Need
+
+The complex gravitational fields of irregular bodies, such as asteroids and comets, are often modeled using polyhedral gravity models as they provide an analytic solution for the computation of the gravitational potential, acceleration (and second derivative) given a mesh of the body [@tsoulis2012analytical;@tsoulis2021computational]. The computation of the gravitational potential and acceleration is a computationally expensive task, especially for large meshes, which can however benefit from parallelization either over computed target points for which we seek potential and acceleration or over the mesh. Thus, a high-performance implementation of a polyhedral gravity model is desirable. 
```

Comment:
Included now:

- Why other models are failing/ unsuitable for the use case (spherical harmonics, mascon models)
- Shifted the focus more on the whole implementation's advantages since we're lacking time measurements for other implementations. Other open source impl. are either a pain to set up. Or papers describing the performance of their employed polyhedral (most of the time, just a sidenote as the model isn't the core of the paper) models lack the required information about the full methodology. I've often seen something like: "Evaluation for one point was expensive and took several seconds," but then they lacked information about the employed strategy (Werner, Tsoulis, etc.), the employed hardware, and the employed mesh.